hera_cal/__init__.py

```diff
@@ -1,9 +1,7 @@
 # -*- coding: utf-8 -*-
-# Copyright 2018 the HERA Project
```

Comment:
My understanding  is that typically you only have to put the oldest year for the copyright and then it's implicitly carried forward, but now that they've been changed we can keep them as is.

Reply:
I think you're right, but for some reason we did it last year...